### ISSUE TITLE
Why is HANDBLACKHOLE by default false?

### ISSUE THREAD

**Issue by Machiry** [SVF-tools/SVF#94]
Is this a sound choice?
This causes the `pointsTo` of `argv` of `main` function to be empty. IMO, this is not correct. We should add a dummy or a black hole node. 
I know that this flag could be turned on, but was wondering whether the value should be by default `true`?

**Comment by yuleisui** [comment]
Yes, you can get more sound results (e.g., arguments of main and point to int casting) when "-blk" is turned. However, it may introduce more imprecise results as a side-effect, which is usually not favorable for bug detection purpose. We set the default one as false at the moment.
